code/lib/instrumenter/src/instrumenter.ts

```diff
@@ -333,7 +333,8 @@ export class Instrumenter {
   // returns the original result.
   track(method: string, fn: Function, args: any[], options: Options) {
     const storyId: StoryId =
-      args?.[0]?.__storyId__ || global.window.__STORYBOOK_PREVIEW__?.urlStore?.selection?.storyId;
+      args?.[0]?.__storyId__ ||
+      global.window.__STORYBOOK_PREVIEW__.selectionStore.selection.storyId;
```

Comment:
What's the difference here between `selectionStore` and `urlStore`?

Reply:
We renamed the variable 😬

Reply:
It's the same. @tmeasday reworked the PreviewWeb interface and split/refactored some code to make it easier for React Native projects, which led to renaming `urlStore` to `selectionStore`. You can see more about it here https://github.com/storybookjs/storybook/pull/19623